### ISSUE TITLE
[FEATURE REQUEST]  Automatically clear cookies automatically from open tabs

### ISSUE THREAD

**Issue by cannyboy** [mrdokenny/Cookie-AutoDelete#925]
**Please describe what you need support with.** 

Is there a way to automatically delete cookies after a set period .. say 2 seconds .. after the page has loaded? Not when tab closed or domain changed, but just after it has loaded. I can see "Clean, including open tabs" in the pop-up, but can't see a way to do this automatically.

**Describe the solution you'd like** 

An option in the settings to auto-delete open tabs (or all cookies), after a set period, without the need for closing tabs or changing domain.

**Comment by kennethtran93** [comment]
This is a feature request at this point - and it will most likely fall in line with a blacklist feature.  Normally cookies should still be there when the site is up, but I understand those that want them removed while tabs are still open.  May break some sites, but that's ultimately up to the user I guess.

**Comment by hackel** [comment]
I think this would be a very useful privacy enhancement, possibly a dupe of #134.

Use case: I have one container where Google is allowed to set cookies and is whitelisted, which is only used for Google services that require a login.  For general Google searches, I open them in the default container where Google is not whitelisted.  The problem is that I often leave a ton of tabs open, including google.com, so CAD never actually deletes those cookies in the default container, thereby allowing Google to track everything I search.

Basically what I want is the ability to add an expression on a container to an autoclean list, as described here, which erases them automatically after a few seconds.
